Mobile/starter_project_matador/lib/features/example/presentation/widgets/introduction.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class Introduction extends StatefulWidget {
+  late String topic;
+  late String content;
+  late int currPage;
+  late int totalPages;
+  Introduction(this.topic, this.content, this.currPage, this.totalPages,
+      {super.key});
+
+  @override
+  State<Introduction> createState() => _IntroductionState();
+}
+
+class _IntroductionState extends State<Introduction> {
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    return Container(
+      width: size.width,
+      padding: const EdgeInsets.only(top: 25, left: 50, right: 50),
+      decoration: const BoxDecoration(
+        color: Colors.white,
+        boxShadow: [
+          BoxShadow(
+            color: Color.fromARGB(166, 11, 59, 104),
```

Comment:
same comment for colors as above
and it could apply to other things as well, like maybe text style